<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArticulationsEditorWidget</class>
 <widget class="QWidget" name="ArticulationsEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Articulations Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeView" name="articulationTreeView">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_4">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="selectANodeLabel">
         <property name="text">
          <string>Select a Node</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="2">
          <widget class="QLabel" name="rollLabel">
           <property name="text">
            <string>Roll</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="pitchLabel">
           <property name="text">
            <string>Pitch</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="zOffsetLabel">
           <property name="text">
            <string>Z Offset</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="xOffsetLabel">
           <property name="text">
            <string>X Offset</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QDoubleSpinBox" name="zOffsetSpin">
           <property name="minimum">
            <double>-9999.989999999999782</double>
           </property>
           <property name="maximum">
            <double>9999.989999999999782</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QDoubleSpinBox" name="headingSpin">
           <property name="suffix">
            <string/>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="headingLabel">
           <property name="text">
            <string>Heading</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSlider" name="rollSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="pitchSpin">
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QSlider" name="headingSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="rollSpin">
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="yOffsetLabel">
           <property name="text">
            <string>Y Offset</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="yOffsetSpin">
           <property name="minimum">
            <double>-9999.989999999999782</double>
           </property>
           <property name="maximum">
            <double>9999.989999999999782</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QDoubleSpinBox" name="xOffsetSpin">
           <property name="minimum">
            <double>-9999.989999999999782</double>
           </property>
           <property name="maximum">
            <double>9999.989999999999782</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="pitchSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QDoubleSpinBox" name="xScaleSpin">
           <property name="maximum">
            <double>999.990000000000009</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="xScaleLabel">
           <property name="text">
            <string>X Scale</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="yScaleLabel">
           <property name="text">
            <string>Y Scale</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="zScaleLabel">
           <property name="text">
            <string>Z Scale</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QDoubleSpinBox" name="zScaleSpin">
           <property name="maximum">
            <double>999.990000000000009</double>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="yScaleSpin">
           <property name="maximum">
            <double>999.990000000000009</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="sequenceActiveCheck">
         <property name="text">
          <string>Active</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="switchIndexLabel">
         <property name="text">
          <string>Index:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="switchIndexCombo">
         <item>
          <property name="text">
           <string>0 [TLOff]</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 [TLOn]</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
